### ISSUE TITLE
Multiple Alarms at same time, can not be canceled. 

### ISSUE THREAD

**Issue by schubidoo** [theScrabi/Terminightor#7]
I recently had several cases where the app went into "ringing frenzy", which sounded like it was playing several ringtones simultaneously. The alarm sounds normal for a fraction of a second, then it sounds like more and more alarms are started.
Stopping the alarm was also not possible.

I believe in all cases I had used an 'old" alarm, which had been disabled for a long time, and then re-enabled it. It seems that then manybalarm events are generated somehow (the last part is speculation).

**Comment by theScrabi** [comment]
Have you tried restart :P
Serious now, it may be. If you recently installed an update of the app that might have caused it. The latest update changed the way of how alarms are handled completely, so there might be some migration problem.

**Comment by theScrabi** [comment]
Um no. It would be better if you just try to reinstall the app completely.

**Comment by schubidoo** [comment]
Is it possible that this also messed up the alarms from the "regular" alarm all of Android? Because my phone did not wake me the day after I encountered that problem. Even with the normal alarm.

I then discovered that the alarm icon had disappeared. Opening that (standard app) alarm, enabling and disabling it, brought the icon back.
Luckily it was a weekend.

**Comment by theScrabi** [comment]
I don't think so, but I also had bad experience with android build in alarm. That PendingIntent thing of android all looks kind of messy to me so yea. But its impossible to interferer with alarms of other apps. Those PendingIntents you setup to make your app do something at a certain time only work for the app who set them up. As far as I know, they are strictly divided between apps.
